ToolsQA/Pages/WikiPage.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+using ToolsQA.Selenium_Basics;
+
+namespace ToolsQA.Pages
+{
+    class WikiPage
+    {
+        [FindsBy(How = How.CssSelector, Using = "[href*='/w/index.php?title=Specjalna:Zaloguj&returnto=Wikipedia%3AStrona+g%C5%82%C3%B3wna']")]
+        private IWebElement _loginButton;
```

Comment:
If it was not readonly property (in this case it could be) you should not prefix it with _